samples/bookinfo/policy/mixer-rule-cnn-redis-quota-fixed-window.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: config.istio.io/v1alpha2
+kind: handler
+metadata:
+  name: redishandler
+  namespace: istio-system
+spec:
+  compiledAdapter: redisquota
+  params:
+    redisServerUrl: redis-release-master:6379
+    connectionPoolSize: 10
+    quotas:
+    - name: requestcountquota.instance.istio-system
+      maxAmount: 500
+      validDuration: 30s
+      rateLimitAlgorithm: FIXED_WINDOW
+      # The first matching override is applied.
+      # A requestcount instance is checked against override dimensions.
+      overrides:
+      # The following override applies to 'istio-egressgateway'.
+      - dimensions:
+          destination: istio-egressgateway
+        maxAmount: 50
+---
+apiVersion: config.istio.io/v1alpha2
+kind: instance
+metadata:
+  name: requestcountquota
+  namespace: istio-system
+spec:
+  compiledTemplate: quota
+  params:
+    dimensions:
+      destination: destination.service.name| destination.labels["app"] | "unknown"
```

Comment:
Can we use some value for "cnn.com" here? Maybe `destination.service.host` or whatever else is available.

Reply:
Thanks fixed.. using request.host